go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/ElrondNetwork/elrond-go-logger v1.0.10
 	github.com/ElrondNetwork/elrond-go-p2p v1.0.5
 	github.com/ElrondNetwork/elrond-go-storage v1.0.4
-	github.com/ElrondNetwork/elrond-vm-common v1.3.30
+	github.com/ElrondNetwork/elrond-vm-common v1.3.31-0.20221209141645-09c6be340647
```

Comment:
do not forget about proper release

Reply:
will create a vmcommon release after the testing is done